drivers/interrupt_controller/intc_gic.c

```diff
@@ -115,7 +115,18 @@ unsigned int arm_gic_get_active(void)
 {
 	int irq;
 
-	irq = sys_read32(GICC_IAR) & 0x3ff;
```

Comment:
should make `irq` `unsigned int` if we do this.

Reply:
Ok, changed